messages/create_scratch.md

```diff
@@ -4,19 +4,24 @@ Create a scratch org.
 
 # description
 
-There are two ways to create a scratch org: either specify a definition file that contains the options or use the --edition/--snapshot flag to specify the one required option.
+There are four ways to create a scratch org:
 
-For either method, you can also use these flags; if you use them with --definition-file, they override their equivalent option in the scratch org definition file:
+    * Specify a definition file that contains the scratch org options.
+    * Use the --edition flag to specify the one required option; this method doesn't require a defintion file.
+    * Use the --snapshot flag to create a scratch org from a snapshot. Snapshots are a point-in-time copy of a scratch org that you create with the "sf org create snapshot" command.
+    * Use the --source-org flag to create a scratch org from an org shape. Org shapes mimic the baseline setup of a source org without the extraneous data and metadata; you create an org shape with the "sf org create shape" command.
```

Comment:
Isn't that command the way you create the snapshot? Not the scratch org as is implied by how it's written? Am I confused?